libs/components/router/testing/src/href/fixtures/href-harness-test.component.html

```diff
@@ -6,4 +6,7 @@
     >Link 1</a
   >
   <a data-sky-id="my-href-2" skyHref="https://example.com/test2">Link 2</a>
+  <a data-sky-id="my-href-3" skyHref="https://example.com/{{ baseHref }}"
```

Comment:
When a variable is bound to the attribute, Angular replaces `skyHref="blah"` with `ng-reflect-sky-href="blah"`.